wiki/Interface/zh.md

```diff
@@ -19,9 +24,9 @@ outdated: true
 -   ［11］用户信息旁边的是地图总数，运行时间，以及电脑时显示间。
 -   ［12］用户面板，点击显示用户选项。
 
-## 用户选项
+### 用户选项
 
-<img src="useroptions1.png" title="User Options" alt="User Options" width="280" />
```

Comment:
shouldn't the alt text still be in the markdown..? is this not regressing?

(applies to everywhere else)

Reply:
I assumed it shouldn't be here as it's not translated. Can bring it back if thats not a good way

Reply:
keep title text, untranslated is better than nothing
remove alt text since it's duplicated from title text. block images on the site have their title text printed as a caption already so alt text is only useful when it differs